helix-core/src/main/java/org/apache/helix/cloud/azure/AzureSourceType.java

```diff
@@ -0,0 +1,10 @@
+package org.apache.helix.cloud.azure;
+
+import org.apache.helix.common.cloud.SourceType;
+
+
+public enum AzureSourceType implements SourceType {
```

Comment:
Do you see any specific usage for Azure of fetching source? Otherwise, we can have a generic SourceType instead of AzureSourceType.